app/controllers/users_controller.rb

```diff
@@ -29,6 +29,7 @@ def create
   def update
     respond_to do |format|
       if @user.update_attributes(user_params)
+        @user.postal_addresses.create(address_params)
```

Comment:
Hm, doing so means I'd create a new `address` record every time I update my user profile. What's the benefit of this?
I may not understand the details of this, but I fear we'll just pollute our database with loads of `address` records then - even though we'd only need one per student 🙈 

I'd suggest a different approach: you could just allow the address attributes as `nested_attributes` on the `User` model and include them in the already permitted `user_params`. Doing so, we can stay with only one line like
```ruby
@user.update_attributes(user_params)
```
And just update (or create) a single address record per user.